src/pushi/base/web.py

```
@@ -117,23 +113,91 @@ def on_finish(protocol):
+
+        # bind event handlers
+        protocol.bind("message", on_message)
+        protocol.bind("finish", on_finish)
+        protocol.bind("error", on_error)
+
+        # schedule timeout using netius event loop delay
+        protocol.delay(on_timeout, timeout=WEBHOOK_TIMEOUT)
+
+        try:
+            loop.run_forever()
+        except Exception as exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)